xp/xconnect/DockerFile

```diff
@@ -78,4 +78,4 @@ RUN $solrUrl = 'http://solr:{0}/solr' -f $Env:SOLR_PORT; `
 
 EXPOSE 443
 
-ENTRYPOINT C:\ServiceMonitor.exe w3svc
+ENTRYPOINT /startup.ps1
```

Comment:
deze heet in andere images `C:\Boot.ps1`; hernoem 'm even voor de consistentie

Reply:
👍